src/models/MessageModel.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  Column,
+  CreateDateColumn,
+  Entity,
+  JoinColumn,
+  ManyToOne,
+  OneToMany,
+  PrimaryGeneratedColumn,
+  PrimaryColumn
+} from "typeorm";
+import { Uuid } from "../types";
+import { UserModel } from "./UserModel";
+import { ReportModel } from "./ReportModel";
+
+@Entity("Message")
+export class MessageModel {
+  @PrimaryColumn("uuid")
+  id: Uuid;
+
+  // @ManyToOne(() => UserModel, (user) => user.sentMessages)
+  // @JoinColumn({ name: "sender" })
+  // sender: UserModel;
+
+  // @ManyToOne(() => UserModel, (user) => user.receivedMessages)
+  // @JoinColumn({ name: "receiver" })
+  // receiver: UserModel;
+
+  // @Column()
+  // content: string;
+
+  // @CreateDateColumn({ type: "timestamptz" })
+  // timestamp: Date;
```

Comment:
Why are we commenting these out? Is it because we're storing messages in Firestone? If this won't be used we should just delete the commented section.

Reply:
Yes, that's the reason. I plan on building the actual messages model later